packages/components/src/components/Flyout/Flyout.tsx

```diff
@@ -62,11 +62,16 @@ export const FlyoutTrigger = (
 FlyoutTrigger.displayName = 'Flyout.Trigger';
 
 export type FlyoutContentProps = {
+    /**
+     * Add a shadow to the flyout
+     * @default "medium"
+     */
+    shadow?: 'none' | 'medium' | 'large';
     /**
      * Add rounded corners to the flyout
-     * @default true
```

Comment:
the default used to say true but it was actually undefined previously, I assume this was an oversight and I now set the default to medium (same as true before)